### PR TITLE
Added VQA model

### PR DESCRIPTION
**Pull request by vpn1997**
Added VQA in model zoo.

### CODE REVIEW

File: ide/static/js/netLayout_vertical.js [hunk 1]
```diff
@@ -30,15 +30,20 @@ the algorithm checks if the location at that depth is occupied by any other node
...
 assigned */
 function allocatePosition(layerId, preferredPosition){
+  
   if (!map.hasOwnProperty(preferredPosition[1])) {
     map[preferredPosition[1]] = [];
   }
+  map[0]=[];
   let positionsY = map[preferredPosition[1]];
-  if (positionsY.indexOf(preferredPosition[0]) != -1) { // If X position is taken
+   if (positionsY.indexOf(preferredPosition[0]) != -1) { // If X position is taken
```
Comment (virajprabhu) [inline ide/static/js/netLayout_vertical.js]: Delete space

File: ide/static/js/netLayout_vertical.js [hunk 2]
```diff
@@ -30,15 +30,20 @@ the algorithm checks if the location at that depth is occupied by any other node
 along the X direction, if yes, the closest available right or left position is 
 assigned */
 function allocatePosition(layerId, preferredPosition){
+  
   if (!map.hasOwnProperty(preferredPosition[1])) {
     map[preferredPosition[1]] = [];
   }
+  map[0]=[];
```
Comment (virajprabhu) [inline ide/static/js/netLayout_vertical.js]: Why is this required? Also, add spacing to make style consistent.

File: ide/static/js/netLayout_vertical.js [hunk 3]
```diff
@@ -30,15 +30,20 @@ the algorithm checks if the location at that depth is occupied by any other node
...
   let positionsY = map[preferredPosition[1]];
-  if (positionsY.indexOf(preferredPosition[0]) != -1) { // If X position is taken
+   if (positionsY.indexOf(preferredPosition[0]) != -1) { // If X position is taken
     let temp = preferredPosition[0], i = 2;
     while (1) { // eslint-disable-line
+      
+
```
Comment (virajprabhu) [inline ide/static/js/netLayout_vertical.js]: Delete newline

File: package.json [hunk 4]
```diff
@@ -41,6 +41,6 @@
     "file-loader": "^0.8.5",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.13.1"
+    "webpack": "^1.15.0"
```
Comment (virajprabhu) [inline package.json]: Is this required?

Comment (vpn1997) [reply package.json]: I dont know for sure,I didn't specified the version while installing webpack.
Can you  review the PR, i have removed the unnecessary changes.
